packages/select/helper-text/index.tsx

```diff
@@ -0,0 +1,130 @@
+// The MIT License
+//
+// Copyright (c) 2019 Google, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import * as React from 'react';
+import classnames from 'classnames';
+import {MDCSelectHelperTextAdapter} from '@material/select/helper-text/adapter';
+import {MDCSelectHelperTextFoundation} from '@material/select/helper-text/foundation';
+
+export interface SelectHelperTextProps extends React.HTMLProps<HTMLParagraphElement> {
+  persistent?: boolean;
+  setHelperTextFoundation?: (foundation?: MDCSelectHelperTextFoundation) => void;
+}
+
+interface ElementAttributes {
+  'aria-hidden'?: boolean | 'false' | 'true';
+  role?: string;
+};
+
+interface SelectHelperTextState extends ElementAttributes {
+  classList: Set<string>;
+};
+
+export class SelectHelperText extends React.Component<SelectHelperTextProps, SelectHelperTextState> {
+  foundation?: MDCSelectHelperTextFoundation;
+
+  state: SelectHelperTextState = {
+    'aria-hidden': undefined,
```

Comment:
spoke offline - set to undefined since foundation should programmatically set this. Also need to define this to be of type SelectHelperTextState